src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -4,30 +4,28 @@
  */
 
 import * as vscode from 'vscode'
-import * as codewhispererClient from '../../client/codewhisperer'
 import * as fs from 'fs-extra'
 import { DependencyGraph } from '../dependencyGraph/dependencyGraph'
-import { BMDocument, performBM25Scoring } from './rankBm25'
-import { getRelevantFilesFromEditor, isRelevant } from './editorFilesUtil'
+import { BM25Document, BM25Okapi } from './rankBm25'
+import { isRelevant } from './editorFilesUtil'
 import { ToolkitError } from '../../../shared/errors'
-import { supplemetalContextFetchingTimeoutMsg } from '../../models/constants'
+import { crossFileContextConfig, supplemetalContextFetchingTimeoutMsg } from '../../models/constants'
 import { CancellationError } from '../../../shared/utilities/timeoutUtils'
+import { CodeWhispererSupplementalContextItem } from './supplementalContextUtil'
 
 const crossFileLanguageConfigs = ['java']
 interface Chunk {
     fileName: string
     content: string
     nextContent: string
+    score?: number
 }
-const chunkSize = 10
-const chunkCount = 60
-const topK = 3
 
 export async function fetchSupplementalContextForSrc(
     editor: vscode.TextEditor,
     dependencyGraph: DependencyGraph,
     cancellationToken: vscode.CancellationToken
-) {
+): Promise<CodeWhispererSupplementalContextItem[] | undefined> {
```

Comment:
You are declaring this function as an async function, but inside this function there is no await. Is this intentional?

Reply:
Nope, there is an await at L34  https://github.com/aws/aws-toolkit-jetbrains/blob/main/jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererFileContextProviderTest.kt